src/contributors/MetricsTableContributor.ts

```diff
@@ -287,19 +296,20 @@ export class MetricsTableContributor extends Contributor {
         const metricsTable: MetricsTable = { data: [], header: [] };
         // att the end we setHeaders
         for (const value of columnsOrder) {
-            metricsTable.header.push({ title: value.collection, subTitle: value.field, metric: value.metric });
+            metricsTable.header.push({ title: value.collection, subTitle: value.field, metric: value.metric, rowfield: value.termfield });
```

Comment:
what is the rowfield added to header ?

Reply:
it is added in order to distinguish for the same collection, which column is calculated for what termfield